src/main.rs

```diff
@@ -147,14 +148,38 @@ fn main() {
     let mut chat_input_enabled = false;
 
     'running: for tick in 0.. {
-        // Get player pos
+        // Get Health and Energy
+        let (current_health, max_health) = client.current::<comp::Health>().map_or((0, 0), |health| (health.current(), health.maximum()));
+        let (current_energy, max_energy) = client.current::<comp::Energy>().map_or((0, 0), |energy| (energy.current(), energy.maximum()));
+
+        // Invite Logic
+		let (inviter_uid, invite_kind) = if let Some((inviter_uid, _, _, invite_kind)) = client.invite() {
+    		(Some(inviter_uid), Some(invite_kind))
+		} else {
+    		(None, None)
+		};
+		
+		//Get entity username from UID
```

Comment:
Funky indentation here?

Reply:
There's still a lot of strange indentation issues
![image](https://user-images.githubusercontent.com/3268430/113360054-f145b700-9340-11eb-9e8b-45c7171a7e3a.png)

Reply:
I recommend just running `cargo fmt` on the whole thing